python/observability-online-eval/livekit/livekit-openai.py

```
@@ -18,21 +18,29 @@
-    if event == "maxim.trace.started":
+
+def on_event(event: str, data: dict):
+    if event == "maxim.session.started":
+        session_id = data["session_id"]
+        logger.session({"id": session_id, "name": "custom session name"})
+    elif event == "maxim.trace.started":
         trace_id = data["trace_id"]
-        trace = data["trace"] 
-        logging.debug(f"Trace started - ID: {trace_id}", extra={"trace": trace})       
+        trace = data["trace"]
+        logging.debug(f"Trace started - ID: {trace_id}", extra={"trace": trace})
```

Ruff (0.11.9)

`debug()` call on root logger

(LOG015)